storm-core/src/clj/org/apache/storm/daemon/worker.clj

```diff
@@ -738,6 +738,7 @@
                     (run-worker-shutdown-hooks worker)
 
                     (.removeWorkerHeartbeat (:storm-cluster-state worker) storm-id assignment-id (long port))
+                    (.removeWorkerBackpressure (:storm-cluster-state worker) storm-id assignment-id (long port))
```

Comment:
I tested submiting topologies and killing them and did not see issues do to this added line, but to those with more knowledge of backpressure logic please let me know if you can think of issues.